src/app/shared/search/search-filters/search-filters.component.html

```diff
@@ -1,7 +1,8 @@
-<h3>{{"search.filters.head" | translate}}</h3>
+<label class="h3">{{filterLable+'.filters.head' | translate}}</label>
```

Comment:
This change is also throwing accessibility warnings in WAVE (again the warning is this is an "orphaned form label"). I don't understand why we need to change this from an `<h3>` to a `<label class="h3">`. Could you either change this back to an `<h3>` or explain why this change was made? We need to fix the accessibility here before we can merge.

Reply:
1 accessibility violation was detected
[
  {
    id: 'heading-order',
    impact: 'moderate',
    description: 'Ensures the order of headings is semantically correct',
    helpUrl: 'https://dequeuniversity.com/rules/axe/4.7/heading-order?application=axeAPI',
    nodes: 1,
    html: [ '<h3 _ngcontent-dspace-angular-c275="">Discover</h3>' ]
  }
]